independent-publisher-connectors/AADInvitationManager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,193 @@
+          }
+        },
+        "summary": "Create invitation",
+        "operationId": "CreateInvitation",
+        "description": "Create an invitation for an external user.",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "SendInvitationMessage": {
+                  "type": "boolean",
+                  "description": "Indicates whether an email should be sent to the user being invited or not.",
+                  "default": false
+                },
+                "invitedUserEmailAddress": {
+                  "type": "string",
+                  "description": "The email address of the user being invited."
+                },
+                "inviteRedirectUrl": {
+                  "type": "string",
+                  "description": "The URL the user should be redirected to once the invitation is redeemed."
+                },
+                "invitedUserMessageInfo": {
+                  "type": "object",
+                  "description": "The invitedUserMessageInfo object allows you to configure the invitation message.",
+                  "properties": {
+                    "ccRecipients": {
+                      "$ref": "#/definitions/recipient"
+                    },
+                    "customizedMessageBody": {
+                      "type": "string",
+                      "description": "Customized message body you want to send if you don't want the default message."
+                    },
+                    "messageLanguage": {
+                      "type": "string",
+                      "description": "The language you want to send the default message in. If the customizedMessageBody is specified, this property is ignored, and the message is sent using the customizedMessageBody. The language format should be in ISO 639. The default is en-US."
+                    }
+                  }
+                }
+              }
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "emailAddress": {
+      "type": "object",
+      "properties": {
+        "address": {
+          "type": "string",
+          "description": "The email address of the person or entity."
+        },
+        "name": {
+          "type": "string",
+          "description": "The display name of the person or entity."
+        }
+      }
+    },
+    "recipient": {
+      "type": "object",
+      "properties": {
+        "emailAddress": {
+          "$ref": "#/definitions/emailAddress"
+        }
+      }
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "undefined": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.windows.net/common/oauth2/authorize",
+      "tokenUrl": "https://login.windows.net/common/oauth2/authorize",
+      "scopes": {}
```

Comment:
Hi @Laskewitz I hope you are doing well.
Apologies for the delay in the process.
Can you please add the scopes that you are targeting the Graph API for, and be more specific?
Thank you very much.